src/plugins/es_ui_shared/README.md

```diff
@@ -0,0 +1,33 @@
+## ES UI shared modules
+
+This plugin contains reusable code in the form of self-contained modules
+(or libraries). Each of these modules exports a set of functionality
+relevant to the domain of the module.
+
+## Files and folders overview
+
+- `./public` | `./server`. Folders for grouping server or public code according to the Kibana plugin pattern.
+- `./__packages_do_not_import__` is where actual functionality is kept. This enables modules more control over what functionality is directly exported and prevents parts of modules to be depended on externally in unintended ways.
```

Comment:
minor grammar note: "to be **dependent**" or "to **depend**". 
Also I feel like a word is missing here? "to be dependent on externally <missing_word> in unintended ways"